### DISCUSSION TITLE
为什么layout在nextjs的ssr使用会报错

### DISCUSSION THREAD

**Discussion by Hao-yiwen** [ant-design/ant-design#46410] [Q&A]
<img width="939" alt="截屏2023-12-12 20 42 18" src="https://github.com/ant-design/ant-design/assets/42827938/fdec167a-60d8-4c2b-a4e0-940d50def2c3">

**Comment by Hao-yiwen** [comment]
谁能帮帮我 之前一直写csr 刚开始写ssr 就遇到这个问题了 有些懵逼

**Comment by Hao-yiwen** [comment]
<img width="672" alt="image" src="https://github.com/ant-design/ant-design/assets/42827938/520906cf-7efa-47fd-932a-12907dd290bf">
报错的地方在这里

**Comment by Hao-yiwen** [comment]
<img width="780" alt="截屏2023-12-12 20 48 33" src="https://github.com/ant-design/ant-design/assets/42827938/89152bc7-f31b-420f-a8f1-9ed9deac8ae3">

所以layout组件本身就是不支持 服务端渲染的吗？ 能不能再文档标注一下啊

**Comment by yoyo837** [comment]
提供复现吧

**Reply by Hao-yiwen** [reply]
已经换写法了 具体就是在服务端渲染方式下使用layout组件就会报错

**Comment by zombieJ** [comment]
`_payload` `_init` 这个看起来应该不是 antd 的代码，更像是什么数据框架里的东西被直接作为 React 节点放进去了。
